stock_kardex_report/models/stock_kardex_report.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class StockKardexReport(models.Model):
+    _name = "stock.kardex.report"
+    _description = "This model creates a kardex report for stock moves"
+    _order = "date desc"
+
+    move_id = fields.Many2one("stock.move", readonly=True)
+    product_id = fields.Many2one("product.product", readonly=True)
+    product_uom_id = fields.Many2one("uom.uom", readonly=True)
```

Comment:
Please add meaningful strings for report column names:

![image](https://user-images.githubusercontent.com/19529533/81264307-777a5000-9041-11ea-83b7-f07073a80f56.png)

Reply:
Changed column names in new commit